### PR TITLE
[spec] Fixed typos and missed words in Chapter 2, 5, and 6.

### PR DESCRIPTION
**Pull request by jzhou76**
Fixed several typos and missed words in Chapter 2, 3 and 5 of the Language Specification.

### CODE REVIEW

File: spec/bounds_safety/interoperation.tex [hunk 1]
```diff
@@ -1113,13 +1113,13 @@ \section{Bounds-safe interfaces for generic functions and struct types}
 Here are bounds safe-interfaces for other C standard library
 functions:
 \begin{lstlisting}
-itype_for_any(T) void *malloc(size_t) :
+itype_for_any(T) void *malloc(size_t size) :
```
Comment (sulekhark) [inline spec/bounds_safety/interoperation.tex:1116]: Could you please add a semicolon at the end of the line below (1117)?

File: spec/bounds_safety/void-ptr-replacements.tex [hunk 2]
```diff
@@ -21,7 +21,7 @@ \section{Introduction}
 \begin{itemize}
 \item As handles that hide details of API implementations.  An
 API may provide a handle that is typed as \uncheckedptrvoid\ for 
-users the API, with the API implementation using a pointer to an actual
+using the API, with the API implementation using a pointer to an actual
```
Comment (sulekhark) [inline spec/bounds_safety/void-ptr-replacements.tex:24]: using ->  users of

**Review by sulekhark — COMMENTED** [review]
Thanks! If you could address my two minor comments, I will be happy to merge these fixes.

**Comment by jzhou76** [comment]
> Thanks! If you could address my two minor comments, I will be happy to merge these fixes.

Fixed. Thanks for the feedbacks.

**Review by sulekhark — APPROVED** [review]
LGTM! Thank you!

**Comment by sulekhark** [comment]
Merged the fixes made by Jie Zhou for typos in the spec.

**Pull request merged**